src/js/components/modals/JobStopRunModal.js

```diff
@@ -35,17 +35,18 @@ class JobStopRunModal extends mixin(StoreMixin) {
 
   handleButtonConfirm() {
     const { selectedItems, jobID } = this.props;
-    // TODO DCOS-8763 introduce support for multiple job run IDs
 
-    if (selectedItems.length === 1) {
-      const jobRunID = selectedItems[0];
+    let i = 0;
+    selectedItems.forEach(function() {
```

Comment:
Now we have to send a network request for every run, which can mean thousands of network requests. I  think this downside is non-trivial. @nLight @pierrebeitz 

Alternatively, we can just change the table type to not confuse the users into thinking that multi-selection is possible.
Also, should we backport this?

Reply:
two folks in the office mentioned that this has been bugging them for quite some time.

Reply:
> two folks in the office mentioned that this has been bugging them for quite some time.

What about the customers?

> i think it makes sense to get rid of the illusion that multi-selection would be possible.

I think so too. I am interested in what @nLight has to say.